packages/devtools_app/lib/src/screens/debugger/program_explorer_model.dart

```diff
@@ -32,12 +30,12 @@ class VMServiceObjectNode extends TreeNode<VMServiceObjectNode> {
   static const packagePrefix = 'package:';
 
   final ProgramExplorerController controller;
-  final String name;
+  final String? name;
```

Comment:
can we make this non-nullable? perhaps defaulting to the empty string at the call site that may be passing null?